client/.eslintrc.json

```diff
@@ -0,0 +1,37 @@
+{
+    "env": {
+        "commonjs": true,
+        "es2021": true,
+        "node": true
+    },
+    "extends": ["eslint:recommended", "airbnb", "prettier"],
+    "parserOptions": {
+        "ecmaVersion": 12
+    },
+    "rules": {
+        "max-len": "off",
+        "no-underscore-dangle": "off",
+        "camelcase": "off", 
```

Comment:
not a question, just an observation: I'm surprised that we're not actually following camelcase

Reply:
Actually, I just copied this file from /api so the first few rules got carried over here. With the rule on, there's only 1 case of it - several follow up PRs are probably going to come next PI to go through these rules and fix violations.